Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contests-filters/ContestFilters.tsx

```diff
@@ -113,13 +130,32 @@ const ContestFilters = ({ onFilterClick }: IContestFiltersProps) => {
         [ loadCategories ],
     );
 
+    const defaultStatusFilterId = useMemo(
+        () => 
+            strategiesAreLoaded && categoriesAreLoaded
+                ? possibleFilters
+                    .filter(f => f.type === DEFAULT_FILTER_TYPE)
+                    .filter(sf => sf.name === DEFAULT_STATUS_FILTER_TYPE)[0].id 
```

Comment:
Are we sure that possibleSorting will always contain a filter with the name DEFAULT_SORT_TYPE?
This call will throw an exception if possibleSorting doesn't contain the filter

Reply:
Same here as above - optional chaining operator used.